server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -259,6 +259,22 @@ public void testUnknownGeoShapeSetting() throws Exception {
             .hasHTTPError(BAD_REQUEST, 4000);
     }
 
+    @Test
+    public void testSettingsNotApplicableToBkdTreeIndexes() {
+        Asserts.assertSQLError(() -> execute("create table test (col1 geo_shape INDEX using BKDTREE with (precision='1m'))"))
+            .hasMessageContaining("The parameters precision, tree_levels, and distance_error_pct are not applicable to BKD tree indexes.")
+            .hasPGError(INTERNAL_ERROR)
+            .hasHTTPError(BAD_REQUEST, 4000);
+        Asserts.assertSQLError(() -> execute("create table test (col1 geo_shape INDEX using BKDTREE with (distance_error_pct='0.25'))"))
+            .hasMessageContaining("The parameters precision, tree_levels, and distance_error_pct are not applicable to BKD tree indexes.")
+            .hasPGError(INTERNAL_ERROR)
+            .hasHTTPError(BAD_REQUEST, 4000);
+        Asserts.assertSQLError(() -> execute("create table test (col1 geo_shape INDEX using BKDTREE with (tree_levels=2))"))
+            .hasMessageContaining("The parameters precision, tree_levels, and distance_error_pct are not applicable to BKD tree indexes.")
+            .hasPGError(INTERNAL_ERROR)
+            .hasHTTPError(BAD_REQUEST, 4000);
+    }
```

Comment:
Please check https://github.com/crate/crate/pull/14942#discussion_r1370380600

Reply:
Done. I replied in the thread you mentioned.